Templates/Applications/Office365.json

```diff
@@ -815,7 +841,15 @@
       [
         {
           "name": "static:label",
-          "content": "IdP Signing Certificate"
+          "content": "IdP Signing Certificate",
+          "inline": true,
+          "pad": ":small"
+        },
+        {
+          "name": "static:color",
+          "content": "Required",
+          "color": "destructive",
+          "inline": true
```

Comment:
This is not the correct required label.  You should probably use `other:importance`.  There are examples in the data store templates.